modules/openapi-generator/src/main/resources/python/model_test.mustache

```diff
@@ -5,13 +5,15 @@
 from __future__ import absolute_import
 
 import unittest
+import datetime
+from io import StringIO
 
 {{#models}}
 {{#model}}
 import {{packageName}}
 from {{modelPackage}}.{{classFilename}} import {{classname}}  # noqa: E501
 from {{packageName}}.rest import ApiException
-
+from test import *
```

Comment:
If we go with the Option1 type of model testing, can we remove this?
It was only included so we could access `TestCategory.make_instance` for Option2

Reply:
Done